Tutorials/whatiswot/8-JSON_Schema_in_Practice_3/8-JSON-Schema-3.txt

```diff
@@ -0,0 +1,44 @@
+In the previous videos of JSON Schema, we have shown the features and basic implementations of JSON Schema. In this video, we will deliver the actual validation process with libraries and examples.
+
+Throughout this video, we will mainly focus on validation with the JavaScript library AJV and the Python library          jsonschema, but there are many other libraries in different programming languages and tools to validate JSON Schema. 
+
+Here are some of the popular ones in different programming languages.  There are many more validators. For further information, you can check the JSON schema website.
```

Comment:
For the links like the JSON Schema website, we should say: check out the json schema website via the link in the video description

Reply:
This is still not fixed since it doesnt say video description. Also, we should say "JSON Schema Homepage" (not part of the original feedback)